sdk/communication/azure-communication-chat/src/main/java/com/azure/communication/chat/ChatClientBuilder.java

```diff
@@ -221,8 +221,9 @@ private HttpPipeline createHttpPipeline(HttpClient httpClient,
                                             List<HttpPipelinePolicy> additionalPolicies) {
 
         List<HttpPipelinePolicy> policies = new ArrayList<HttpPipelinePolicy>();
-        policies.add(authorizationPolicy);
         applyRequiredPolicies(policies);
+        // auth policy is per request, should be after retry
+        policies.add(authorizationPolicy);
```

Comment:
Logging policy should be the last policy. So, auth policy should be added before logging policy.